mousetrap.js

```diff
@@ -1018,4 +1029,4 @@
             return Mousetrap;
         });
     }
-}) (window, document);
+}) (typeof window !== 'undefined' ? window : null, typeof document !== 'undefined' ? document : null);
```

Comment:
oh i see, i never did this before. How does mousetrap works on server side? I can't imagine it working without a browser.